src/components/app/characters/CharacterList.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useCharacters } from '../../../hooks/character';
+import CharacterItem from './CharacterItem';
+import { Link } from 'react-router-dom';
+
+const CharacterList = ({ page }) => {
+  const { loading, characters } = useCharacters(page);
+  if (loading)
+    return <h1>Loading...</h1>;
+
+  const characterElements = characters.map(character => (
+    <li key={character.name}>
+      <Link to={`/characters/${character.name}`}>
+        <CharacterItem
+          name={character.name}
+          image={character.image}
+        />
+      </Link>
+    </li>
+  ));
+
+  return (
+    <ul data-testid="characters">
+      {characterElements}
+    </ul>
+  );
+};
+
+CharacterList.propTypes = {
+  name: PropTypes.string.isRequired,
+  image: PropTypes.string.isRequired
```

Comment:
these don't match the props (page) you have in line 7- you can get rid of name and image and have page: PropTypes.number.isRequired